src/libraries/Microsoft.Extensions.Hosting.Systemd/src/SystemdLifetime.cs

```diff
@@ -46,11 +46,29 @@ public SystemdLifetime(IHostEnvironment environment, IHostApplicationLifetime ap
         private ILogger Logger { get; }
         private ISystemdNotifier SystemdNotifier { get; }
 
+        /// <summary>
+        /// Asynchronously stops and shuts down the host. This method is called from <see cref="IHost.StopAsync(CancellationToken)" />.
+        /// </summary>
+        /// <param name="cancellationToken">
+        /// A cancellation token that indicates when stop should no longer be graceful.
+        /// </param>
+        /// <returns>
+        /// A task that represents the asynchronous stop operation.
+        /// </returns>
         public Task StopAsync(CancellationToken cancellationToken)
         {
             return Task.CompletedTask;
```

Comment:
The added comments do not match what the method does. This is only returning the object that represents a completed task.

Reply:
You added those in one of the commits. I assume they are just a straight backport?

Reply:
Not sure I understand. What commits are you referring to?

Reply:
https://github.com/dotnet/runtime/pull/84917/commits/3013103dc9eaf89e5200ac245945a2c66e91b890

Reply:
Oh. I just noticed the date of this PR.

Yes, I ported them using the tool. They exist this way in MS Docs: https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.hosting.systemd.systemdlifetime.stopasync?view=dotnet-plat-ext-3.1

@gewarren I suppose the documentation of this API needs to say something along the lines of "This API does nothing except returning a completed task". Is this correct?

Reply:
@ViktorHofer we can address it in a separate PR.

Reply:
@carlossanlop Yes the summary could be something like "Returns a task that has already completed."